### PR TITLE
Test Windows build on CircleCI

### LINKED ISSUE #48: CircleCI can test Win 64 builds

### PR DESCRIPTION
**Pull request by bahmutov**
- Closes #48 
- Added Windows CircleCI build
- Shows currently failing pipe from https://github.com/cypress-io/cypress/issues/2181
- Once a fix is implemented can be merged
- moved early halt logic into a Node script because the bash syntax was not working on Windows

### CODE REVIEW

**Comment by flotwig** [comment]
@bahmutov This is unblocked now that Cypress can run in Windows CircleCI, right?

**Comment by bahmutov** [comment]
Are the tests passing on circle windows ? It should be I think

**Comment by flotwig** [comment]
Not passing yet, but it seems to be a failure in copying artifacts, not in Cypress itself

**Comment by bahmutov** [comment]
Windows paths shenanigans?

**Comment by bahmutov** [comment]
@flotwig I have updated the dependency `ok-file` that was checking files - it was getting single-quoted argument on Windows, breaking it. Now Circle is happy again and AppVeyor failures are because it is AppVeyor and fails sometimes to pass env variables!

**Comment by flotwig** [comment]
Appveyor has been failing since 2f2b827 in this branch with the same error, also, that job doesn't seem to exist for any of the commits before that 2f2b827

**Comment by bahmutov** [comment]
ok, I removed recording on AppVeyor, enough is enough

File: package.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
   "author": "Gleb Bahmutov <gleb@cypress.io>",
   "license": "ISC",
   "devDependencies": {
-    "ok-file": "1.4.0"
+    "cypress": "3.8.1",
```
Comment (flotwig) [inline package.json]: does this get removed before or after the PR is merged?
```suggestion
```

**Comment by bahmutov** [comment]
yeah, it is ok to remove hardcoded version

**Pull request merged**